Description: Allow reading time configuration from /etc/writable, as we have it on the phone.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Forwarded: No, that's an Ubuntu specific hack
Bug-Ubuntu: https://launchpad.net/bugs/1227520

---
 profiles/apparmor.d/abstractions/base                     |    1 +
 profiles/apparmor.d/abstractions/ubuntu-browsers.d/java   |    2 ++
 profiles/apparmor/profiles/extras/usr.lib.firefox.firefox |    1 +
 profiles/apparmor/profiles/extras/usr.sbin.sshd           |    1 +
 4 files changed, 5 insertions(+)

--- a/profiles/apparmor.d/abstractions/base
+++ b/profiles/apparmor.d/abstractions/base
@@ -26,6 +26,7 @@
   /etc/locale/**                 r,
   /etc/locale.alias              r,
   /etc/localtime                 r,
+  /etc/writable/localtime        r,
   /usr/share/locale-bundle/**    r,
   /usr/share/locale-langpack/**  r,
   /usr/share/locale/**           r,
--- a/profiles/apparmor.d/abstractions/ubuntu-browsers.d/java
+++ b/profiles/apparmor.d/abstractions/ubuntu-browsers.d/java
@@ -38,6 +38,7 @@
     /etc/lsb-release r,
     /etc/ssl/certs/java/* r,
     /etc/timezone r,
+    /etc/writable/timezone r,
 
     @{PROC}/@{pid}/ r,
     @{PROC}/@{pid}/fd/ r,
@@ -82,6 +83,7 @@
     /etc/lsb-release r,
     /etc/ssl/certs/java/* r,
     /etc/timezone r,
+    /etc/writable/timezone r,
 
     @{PROC}/@{pid}/ r,
     @{PROC}/@{pid}/fd/ r,
--- a/profiles/apparmor/profiles/extras/usr.lib.firefox.firefox
+++ b/profiles/apparmor/profiles/extras/usr.lib.firefox.firefox
@@ -38,6 +38,7 @@
   /tmp/.X[0-9]*-lock r,
 
   /etc/timezone r,
+  /etc/writable/timezone r,
   /etc/wildmidi/wildmidi.cfg r,
 
   # firefox specific
